content/getting-started/onboarding-wizard.md

```diff
@@ -88,13 +91,99 @@ If the **Report Path Prefix** lookup is not available (and is instead a freeform
 {{% /notice %}}
 
-### 4. Name & Finalize New Datasource
+#### 4. Name & Finalize New Datasource
 
 1. Provide a name for your AWS datasource.
 2. Select **Confirm & Finish**.
 
 You can create more datasources using the guides in our [Integrations][3] section.
 
+---
+
+## Configure Azure
+
+There are two ways to set up Azure with CloudWisdom: using the Azure CLI or through the Azure Portal. Virtana recommends using the CLI, as it's much quicker and easier.
+
+### Via Azure Cli
+
+1. Log in to **CloudWisdom**. You are automatically directed to the Onboarding Wizard.
+2. Select **Begin Setup** > **Setup Azure Cloud**.
+3. Select **Continue With Azure CLI**.
+4. Ensure you have [installed the CLI][4] before proceeding.
+5. Select **Continue**.
+6. Open a command line terminal; for Windows, this may be PowerShell.
+7. Input the following command:
+`az login`
+8. Log in to your account and return to the command line.
+9. Input the following command:
+`az account show`
+10. Locate your **tenantId** and copy it.
+11. Paste the Id into the **Tenant Id** field.
+![tenant-id](/images/onboarding-wizard/tenant-id.png)
+12. Select **Continue**.
+13. Run the following command, adding in your id for **subscription-id**:
+`az ad sp create-for-rbac --role "Monitoring Reader" --name CloudWisdomReader --scopes /subscriptions/<subscription-id>`
+14. Copy and paste the `appId` into the **Client ID** field.
+15. Copy and paste the `password` into the **Access Key** field.
+16. Select **Continue**.
+17. Optionally [Enable Guest OS Diagnostic Metrics][5].
+18. Select **Continue**.
+19. Select **Confirm & Finish**.
+
+### Via Azure Portal
+
+1. Log in to **CloudWisdom**. You are automatically directed to the Onboarding Wizard.
+2. Select **Begin Setup** > **Setup Azure Cloud**.
+3. Select **Continue With the Azure Portal**.
+4. **Azure Tab**: Open the [Azure Portal](https://portal.azure.com/) in a separate tab and log in.
+5. **Azure Tab**: Navigate to **App Registrations** via search or **Active Directory** > **App Registrations**.
+5. **CloudWisdom Tab**: Select **Continue**.
+6. **Azure Tab**: Select **+ New registration**.
+7. **Azure Tab**: Input and select the following:
+   - **Name**: `cloudwisdom-integration`
+   - **Supported account types**: Accounts in this organizational directory only - Single tenant
+   - **Redirect URI (optional)**: `https://us.cloudwisdom.virtana.com`
+8. **Azure Tab**: Select **Register**. You are re-routed to your new apps overview page.
+![azure-setup-portal](/images/onboarding-wizard/azure-setup-portal.png)
+9. **CloudWisdom Tab**: Select **Continue**.
+10. **Azure Tab**: Copy and paste the following into the CloudWisdom's Onboarding Wizard:
+   - **Application (client) ID**: maps to Client ID
+   - **Directory (tenant) ID**:  maps to Tenant ID
+![az-port-gif](/images/onboarding-wizard/az-port-gif.gif)
```

Comment:
Would it be terribly difficult to trim the top and bottom of the gif so as not to see the tabs and task bar?